Documentation/devel/stage1-implementors-guide.md

```diff
@@ -96,6 +96,14 @@ For example, it removes the network namespace of a pod.
 * `--debug` to activate debugging
 * UUID of the pod
 
+Versioning
+----------
+
+The stage1 ABI is versioned using an annotation with the name `coreos.com/rkt/stage1/interface-version`.
+If the annotation is not present, rkt assumes the ABI version is 1.
+
+The current version of the ABI is 1.
```

Comment:
Ditto, I would simply say - "The current version of the stage1 interface is 1.".